superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -29,6 +29,12 @@ import { getClientErrorObject } from 'src/utils/getClientErrorObject';
 import { FetchDataConfig } from 'src/components/ListView';
 import { Dashboard } from './types';
 
+interface Filters {
+  col: string;
+  opr: string;
+  value: string;
+}
```

Comment:
Nit: let's make this a shared type with a more specific name (e.g. `CRUDFilterParam`) and put it into `views/CRUD/types.tsx`, as it is obviously used in other components, too.

Reply:
Makes sense will do!